### PR TITLE
SG-22653: Shotgun to ShotGrid

### PR DESCRIPTION
**Pull request by jfboismenu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
 [![Linting](https://img.shields.io/badge/PEP8%20by-Hound%20CI-a873d1.svg)](https://houndci.com)
 
 ## Documentation
-This repository is a part of the Shotgun Pipeline Toolkit.
+This repository is a part of the ShotGrid Pipeline Toolkit.
 
 - For more information about this app and for release notes, *see the wiki section*.
 - For general information and documentation, click here: https://support.shotgunsoftware.com/entries/95441257
-- For information about Shotgun in general, click here: http://www.shotgunsoftware.com/toolkit
+- For information about ShotGrid in general, click here: http://www.shotgunsoftware.com/toolkit
```
Comment (eshokrgozar) [inline README.md:11]: Do you want to change the link here as well or leave it until later?

**Review by eshokrgozar — APPROVED** [review]
Looks good. Just a possible change in the README

File: README.md (hunk 1 above)
Comment (jfboismenu) [reply README.md]: Fixed. Thank you. I forgot to update to your version of the script. Thanks!

**Comment by jfboismenu** [comment]
@mathurf , over to you

**Review by mathurf — APPROVED** [review]
Everything looks good

**Pull request merged**